web/ui_utils.js

```diff
@@ -124,9 +124,7 @@ function scrollIntoView(element, spot, scrollMatches = false) {
   while (
     (parent.clientHeight === parent.scrollHeight &&
       parent.clientWidth === parent.scrollWidth) ||
-    (scrollMatches &&
-      (parent.classList.contains("markedContent") ||
-        getComputedStyle(parent).overflow === "hidden"))
+    (scrollMatches && !parent.classList.contains("pdfViewerContainer"))
```

Comment:
What happens if you add that condition as another || ?
Does that still work? If it does, that's preferable to removing stuff, since we have no way of validating this won't break other things that rely on it.

Reply:
Smart! I'll test and get back to you.

Reply:
Resolved